lib/pages/home/home_view_model.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_survey/pages/home/home_state.dart';
+import 'package:flutter_survey/usecase/get_surveys_use_case.dart';
+
+class HomeViewModel extends StateNotifier<HomeState> {
+  final GetSurveysUseCase _getSurveysUseCase;
+
+  HomeViewModel(this._getSurveysUseCase) : super(const HomeState.init()) {
+    init();
+  }
+
+  Future<void> init() async {
```

Comment:
slightly concerned if this needs to be async 🤔  I think to test, you can just simply call the async use case function and just print out the result whenever it arrives yeah?

Reply:
@sleepylee correct, copy-pasted issue 😆 fixing